vignettes/vrd_vignette.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "vrd_vignette"
+output: rmarkdown::html_vignette
```

Comment:
Update title, something like: "Introduction to caRecall", add authors section (probably use all of our names)
`authors: authors: "Rryan Koenig, Nathan Smith, Mitch Harris"`